snooty/parser.py

```diff
@@ -1005,9 +1005,17 @@ def __init__(
         self.config.substitution_nodes = substitution_nodes
 
         username = pwd.getpwuid(os.getuid()).pw_name
-        branch = subprocess.check_output(
-            ["git", "rev-parse", "--abbrev-ref", "HEAD"], cwd=root, encoding="utf-8"
-        ).strip()
+        try:
+            branch = subprocess.check_output(
+                ["git", "rev-parse", "--abbrev-ref", "HEAD"],
+                cwd=root,
+                encoding="utf-8",
+                stderr=subprocess.PIPE,
+            ).strip()
+        except subprocess.CalledProcessError as err:
+            logger.info("git error getting branch name: %s", err.stderr)
+            branch = "current"
+
```

Comment:
Just a note, would be nice to abstract this from the main parser process eventually. It feels somewhat specific to the MongoDB backend approach we are using.

Reply:
oh YES absolutely!

This is a quirk that we should rethink. But right now, I just want it to not be a *fatal* quirk 😸